### PR TITLE
Portfolio: setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by SlowKingV**
# Headline and menu bar completed!

The first part of the project portfolio required us to build the first section and navbar of the template we chose.

## Requirements
- For this project, you should use the [HTML&CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css)
- The 2 sections you need to build for this project are:

  - the toolbar (or header)
  - the headline section (right after the header)
- In order to layout the elements on the page you should use Flexbox in all 2 sections.
- You don't need to implement any functionality that requires JavaScript, like opening the menu or creating dropdowns.
- You must stick to the design as much as possible (e.g. font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+<body>
+  <div id="head-bar">
+    <span>My Logo</span>
+    <div class="menu-icon">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+  </div>
```
Comment (leonardodiasb) [inline index.html]: - [x] Kindly wrap the toolbar in a `<header>` tag as it is mentioned in the Project Requirements.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <script src="https://kit.fontawesome.com/bf254b5930.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <div id="head-bar">
+    <span>My Logo</span>
```
Comment (leonardodiasb) [inline index.html]: - [x] Logos are usually a `<a>` link to the homepage. Kindly fix that issue.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    <div class="menu-icon">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
```
Comment (leonardodiasb) [inline index.html]: - [x] The hamburger button should be inside a `<nav>` tag. Kindly change that.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          <a href="#"><i class="fa-brands fa-medium"></i></a>
+        </li>
+        <li class="github">
+          <a href="#"><i class="fa-brands fa-github"></i></a>
+        </li>
+        <li class="angellist">
+          <a href="#"><i class="fa-brands fa-angellist"></i></a>
+        </li>
+      </ul>
+    </header>
```
Comment (leonardodiasb) [inline index.html]: - [x] The whole headline is a `<section>`. Kindly change that tag, the `<header>` is the toolbar as mentioned in the Project Requirements.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+  <main>
+    <header>
+      <h1>
+        I&rsquo;m Mariane<br>
+        Glad to see you!
+      </h1>
+      <p>
+        I&rsquo;m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don&rsquo;t hesitate to contact me.
+      </p>
```
Comment (leonardodiasb) [inline index.html]: - [x] You added your sub-title inside the paragraph tag. Kindly separate and style it as the Figma template design. Remember that you can use other levels like `<h2>` tags.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+      <span class="emphasize">LET&rsquo;S CONNECT</span>
```
Comment (leonardodiasb) [inline index.html]: - [x] This should be a button or a link to some form(that will be built in the future) in your portfolio website. Use proper tags.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @SlowKingV, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️
Your CSS style is amazing!!! ✔️

## Aspects to improve ♻️

- [x] Please add a background image just like the Figma template.
- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SlowKingV** [comment]
I have added the background image and fixed all of the optional changes mentioned above.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+        I&rsquo;m Mariane<br>
+        Glad to see you!
+      </h1>
+      <h3>
+        I&rsquo;m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don&rsquo;t hesitate to contact me.
+      </h3>
+      <span class="emphasize no-decoration"><a href="#">LET&rsquo;S CONNECT</a></span>
+      <ul class="social-links">
+        <li class="twitter">
+          <a href="#" title="Twitter"><i class="fa-brands fa-twitter no-decoration"></i></a>
```
Comment (phelian23) [inline index.html:36]: - [x] [Optional] The navbar is overlapping the headline section. Kindly fix it

**Review by phelian23 — APPROVED** [review]
Hi @SlowKingV, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**